modules/nw-sriov-dpdk-example-mellanox.adoc.adoc

```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="example-vf-use-in-dpdk-mode-mellanox_{context}"]
+= Using a virtual function in DPDK mode with a Mellanox NIC
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the SR-IOV Network Operator.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the following `SriovNetworkNodePolicy` object, and then save the YAML in the `mlx-dpdk-node-policy.yaml` file.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: mlx-dpdk-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: mlxnics
+  nodeSelector:
+    feature.node.kubernetes.io/network-sriov.capable: "true"
+  priority: <priority>
+  numVfs: <num>
+  nicSelector:
+    vendor: "15b3"
+    deviceID: "1015" <1>
+    pfNames: ["<pf_name>", ...]
+    rootDevices: ["<pci_bus_id>", "..."]
+  deviceType: netdevice <2>
+  isRdma: true <3>
+----
+<1> Specify the device hex code of the SR-IOV network device. The only allowed values for Mellanox cards are `1015`, `1017`.
+<2> Specify the driver type for the virtual functions to `netdevice`. Mellanox SR-IOV VF can work in DPDK mode without using the `vfio-pci` device type. VF device appears as a kernel network interface inside a container.
+<3> Enable RDMA mode. This is required for Mellanox cards to work in DPDK mode.
++
+[NOTE]
+=====
+See the `Configuring SR-IOV network devices` section for detailed explanation on each option in the `SriovNetworkNodePolicy` object.
+
+When applying the configuration specified in a `SriovNetworkNodePolicy` object, the SR-IOV Operator may drain the nodes, and in some cases, reboot nodes.
```

Comment:
The ISG suggests using "may" only when referring to permission. You could replace it with "might" in this case:

> the SR-IOV Operator might drain the nodes

Reply:
Done.